Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM openjdk:8-jdk-alpine
+
+ARG CACHEBUST=1
```

Comment:
```suggestion
# to make sure that the cache is only used during one day, run docker build --build-arg CACHEBUST=$(date "+%Y-%m-%d")
# that way we should get the latest updates since the release of our base image 
# thx to https://github.com/moby/moby/issues/1996#issuecomment-185872769
ARG CACHEBUST=1
```

I'm aware that such comments tend to make the Dockerfile large&ugly, but without them I fear that without them, in a few weeks it will be hard to remember wtf this was all about